Bug fixes and new keys

HonestLocksmith opened the pull request.

zinongli commented:
God damn! That's a lot of car keys. 

This could benefit from [this](https://github.com/zinongli/KeyCopier/pull/13/commits/0d483d6025374219af5d39e5f76efc9b868dbe02) since you wouldn't want the bottom horizontal line in the way for double sided keys. I will make a few adjustments for some format's clearnace, e.g. RU45 should have a clearance of 5 instead of 8. 

In the mean time, this format has some weird graphics that I'm not sure what caused it.
![Screenshot-20250116-235853](https://github.com/user-attachments/assets/df4d7f69-683b-4e5a-9173-23dac0d342a8)

HonestLocksmith commented:
I've fixed the issue. Lowered it to 6 pin, which is fine, and moved first pin spacing to the right and matched the last pin inch with the added space.

zinongli commented:
Thanks for the fix! I also adjust some format's clearances manually according to their own bitting step depths.

I think I'm also having issue with Y2 if the first pin goes more than 6 because the slanted line on the left of the pin would exceed screen's edge and give weird graphics. 

![Screenshot-20250117-002837](https://github.com/user-attachments/assets/8b6508e8-496e-4113-b116-dad788e1bf0e)

HonestLocksmith commented:
You should keep an eye on my saved files that go with the program. It makes a lot better to understand what I've been doing.

zinongli commented:
Yes I see that you included the data sheets and manufacturer. Very impressive work! I'm not sure if OFW team would like us to show brands names explicitly in the user interface but having it in the saved file should be fine.

HonestLocksmith commented:
fixed y2. Sorry. I knew there was a few last bugs to work out.

zinongli commented:
No it's all good. Bugs happen all the time. Your last Y2 fix commit reverted some clearance fix I made. You might wanna pull the PR branch to stay updated before adding new changes on top of it. But no need to do that this time--I will resolve the conflicts.

zinongli commented:
Looks good to me!
